### DISCUSSION TITLE
A way to ignore following line? (VS Code extension)

### DISCUSSION THREAD

**Discussion by KamilDev** [micromark/micromark#188] [Q&A]
I'm using a Liquid preprocessor on MDX files, so the following will give an error in VS Code:
```liquid
{% include "api-docs-table.liquid", apiDoc: "apiDocs.converter" %}
```
> Could not parse expression with acornMDX ([micromark-extension-mdx-expression:acorn](https://github.com/micromark/micromark-extension-mdx-expression/tree/main/packages/micromark-extension-mdx-expression#could-not-parse-expression-with-acorn))

Is there any way to configure to ignore such errors, for example similar to `@ts-ignore`:
```mdx
{ /* @acorn-ignore */ }
{% include "api-docs-table.liquid", apiDoc: "apiDocs.converter" %}
```

**Comment by remcohaszing** [comment]
No. If you tell your editor that a file is of a certain type, but instead you write a text templating syntax, then your editor won’t understand the syntax.

You’re talking about MDX, but the same problems applies to every other language.

**Comment by KamilDev** [comment]
The ability to ignore specific lines is [not planned](https://github.com/micromark/micromark-extension-mdx-expression/issues/10#issuecomment-2561136387). I've just disabled `mdx.server.enable` in the meantime before exploring other solutions.

**Reply by ChristianMurphy** [reply]
@KamilDev thay is a rather misleading statement.
MDX supports ignores.
What you describe here isn't an ignore.
Not in the sense that prettier, eslint, typescript or any other tool thinks about ignores.
Your file is syntactically incorrect, no tool can process it, and ignore can't fix that.

There are ways to make your file use MDX syntax, or to add your flavor of markdown to remark/mdx, but an ignore isn't the way.
https://github.com/micromark/micromark-extension-mdx-expression/issues/10#issuecomment-2561136387
